pipelines/matrix/src/matrix/pipelines/run_comparison/settings.py

```diff
@@ -0,0 +1,26 @@
+"""Dynamic settings for run_comparison pipeline."""
+
+RUN_COMPARISON_SETTINGS = {
+    "run_comparison": {
+        "inputs": {
+            # # Example entries. Update filepaths as needed.
+            # "auto-kg-release-v0-10-1-072eb02b.transformed_matrix": {
+            #     "filepath": "gs://mtrx-us-central1-hub-dev-storage/kedro/data/releases/v0.10.1/runs/auto-kg-release-v0-10-1-072eb02b/datasets/matrix_transformations/fold_3/transformed_matrix"
+            # },
+            # "auto-kg-release-v0-10-2-fe36a9cb.transformed_matrix": {
+            #     "filepath": "gs://mtrx-us-central1-hub-dev-storage/kedro/data/releases/v0.10.2/runs/auto-kg-release-v0-10-2-fe36a9cb/datasets/matrix_transformations/fold_3/transformed_matrix"
+            # },
+            "test-release-8": {
+                "filepath": "data/test/releases/test-release/runs/test-run-8/datasets/matrix_transformations/fold_0/transformed_matrix"
```

Comment:
We should probably think through how we can set the path above correctly.

It will be a little bit of a chicken and egg problem, as the settings are loaded _before_ the catalog is initialised, so you won't have access to the catalog (I think?).

Probably want to do something like: 

```
RUN_COMPARISON_SETTINGS=resolve_base_path(obj)
```